app/assets/javascripts/admin/views/editions/_form.js

```diff
@@ -265,6 +266,27 @@
         $(this).parent().hide().next().removeClass('if-js-hide');
         e.preventDefault();
       });
+    },
+
+    showImageUploaderIfCustomImage: function showImageUploaderIfCustomImage() {
+      var $imageUploader = $('.image_section .js-show-image-uploader');
+      var $customImage = $('#edition_image_display_option_custom_image');
+      var $radioButtons = $('.image_section input[type=radio]');
+
+      function showOrHideImageUploader() {
+        if ($customImage.prop('checked')){
+          $imageUploader.show();
+        } else {
+          $imageUploader.hide();
+        }
+      }
+      $radioButtons.on('change', showOrHideImageUploader);
+      showOrHideImageUploader();
+
+      $('.image_fields a').on('click', function(e) {
```

Comment:
I think this will only work on click, which wouldn't be accessible to people who need to use a keyboard. `on('submit'` is usually a better choice, but I'm not sure it'll work with links (it's usually used for buttons). May need some experimentation to get working for keyboard users 🤔

Reply:
I think the `on('click'` part has been removed with the removal of the anonymous function

Reply:
Oops, of course! Thanks. I'll resolve this convo 👍